### ISSUE TITLE
Reflection data improvement

### ISSUE THREAD

**Issue by ToniPlays** [peter1745/Coral#2]
Few useful features:
- Method return value
- Method parameter reflection
- Class, field and method attribute types

Thanks Peter, very cool

**Comment by VagueLobster** [comment]
I agree, that these feature requests would be a nice thing to have, but very cool framework nevertheless Peter 👍
